anup_kumar_Postgres/schoolapi/src/controller/marks.ts

```diff
@@ -0,0 +1,59 @@
+import { ValidatePage, ValidId } from "../helper/index";
+import { Context } from "vm";
+import { AllMarks, AddMarks, Topper } from "../services/marks";
```

Comment:
function should start with lower case. Can you also define types/interface for all function props, request body and were ever we are defining variables, functions, objects